### ISSUE TITLE
@kubb/swagger-zod output to single file create redundant imports

### ISSUE THREAD

**Issue by aaronb-reach** [kubb-project/kubb#824]
### What version of `kubb` is running?

kubb/2.6.1 linux-x64 node-v20.11.0

### What platform is your computer?

Linux (but happens on MacOS too)

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

```
{
  "name": "kubb",
  "version": "1.0.0",
  "scripts": {
    "generate": "kubb generate --config kubb.config.js"
  },
  "devDependencies": {
    "@kubb/cli": "^2.6.1",
    "@kubb/core": "^2.6.1",
    "@kubb/swagger": "^2.6.1",
    "@kubb/swagger-ts": "^2.6.1",
    "@kubb/swagger-zod": "^2.6.1"
  }
}
```

### What steps can reproduce the bug?

When outputting to zod schemas in a single file an extra `import { petsSchema } from "./";` is added at the top of the file which is redundant and creates TS compilation errors as `petsSchema` is already defined in the generated file and then exported.

Reproduction of issue: https://github.com/burt202/kubb-test
Issue is within the generated zod file at: https://github.com/burt202/kubb-test/blob/master/generated/zod.ts

TS error: `Individual declarations in merged declarations 'petsSchema' must be all exported or all local`

NOTE: the test repo above recreates the core issue of the extra import, but i did not add TS to it to keep the repo as minimal as possible

To recreate:
- clone repo above
- npm ci
- npm run generate

### How often does this bug happen?

Every time

### What is the expected behavior?

The import at the top of the generated zod file should not be added

### Swagger/OpenAPI file?

Standard Pet Store yml

https://github.com/burt202/kubb-test/blob/master/docs.yml

**Comment by stijnvanhulle** [comment]
Fixed with `2.6.2`

**Comment by aaronb-reach** [comment]
Great, thanks for the quick turnaround!
